runtime/doc/options.txt

```diff
@@ -3267,7 +3267,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 				     >:SignColumn,B:SpellBad,P:SpellCap,
 				     R:SpellRare,L:SpellLocal,-:Conceal,
 				     +:Pmenu,=:PmenuSel,x:PmenuSbar,
-				     X:PmenuThumb")
+				     X:PmenuThumb,q:QuickFixLine")
```

Comment:
I agree with you about the use of `snapshot_util()` and am actually using it now for another test.  What I'm referring to is the output of a failed test using `expect()`.

What it looks like now:

```
Expected:
|*blah blah bla|
Actual:
|*                                        |
|{UNEXPECTED reverse = true:[No Name]                               }|
|message 1                               |
|{UNEXPECTED background = Screen.colors.Green:message 2}       |
|{UNEXPECTED background = Screen.colors.Yellow:message 3}        |
|{UNEXPECTED background = Screen.colors.Fuchsia:^message 4}          |
|{1:~                                       }|
|{1:~                                       }|
|{UNEXPECTED bold = true, reverse = true:nvim://messages }|
|{UNEXPECTED background = Screen.colors.Fuchsia:message 4}         |
```

What I think it should look like:

```
Expected:
|*blah blah bla|
Actual:
|*                                          |
|{2:[No Name]                               }|
|message 1                               |
|{3:message 2}                               |
|{4:message 3}                               |
|{5:^message 4}                               |
|{1:~                                       }|
|{1:~                                       }|
|{6:nvim://messages                         }|
|{5:message 4}                               |

[1] = {bold = true, foreground = Screen.colors.Blue}
UNEXPECTED [2] = {reverse = true}
UNEXPECTED [3] = {background = Screen.colors.Green}
UNEXPECTED [4] = {background = Screen.colors.Yellow}
UNEXPECTED [5] = {background = Screen.colors.Fuchsia}
UNEXPECTED [6] = {bold = true, reverse= true}
```

As someone who isn't an expert in the test suite or with Lua, I think that output is clearer.